modules/registry-operator-default-crd.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-default-crd-{context}"]
+= Image Registry Operator Custom Resource Definition and parameter values
+
+In {product-title}, the `Registry` Operator controls the registry feature. The
+Operator is defined by the `configs.imageregistry.operator.openshift.io` Custom
+Resource Definition (CRD).
+
+////
+The following CRD displays the default configuration
+and explains both the parameters you can configure and valid parameter values:
+
+
+.Default <resource> CRD
+[source,yaml]
+----
+//Include the default CRD for your resource. Make sure that you show the default values in the CRD.
+//For each configurable parameter, add an annotation in the <n> format, where n is the ordered sequence of integers starting with 1.
+----
+//For each annotation, explain the parameter, it’s type (like string, boolean, integer), and acceptable values. If the default value is not a good representation of the typical value, include a good example. If there are other known restrictions on the parameter, share them. The annotation might resemble this example:
+//<1> The size of the volume in `Gi`. This value must be a positive integer, such as `500`. Because this volume deploys to the location that you specified in the `other.parameter` parameter, ensure that sufficient space is available.
+----
+////
+
+.Procedure
+
+If you need to patch the Image Registry Operator CRD, run the following command.
```

Comment:
I'd combine this instruction with the step.
Also, why am I patching it, and what am I patching it with?